benchmarks/source-kontent/.env.template

```diff
@@ -0,0 +1,3 @@
+KONTENT_PROJECT_ID=4d1eb9ae-c95e-00e1-da99-692832fe675c 
```

Comment:
Can you please prefix all environment variables with `BENCHMARK`? ex. `BENCHMARK_KONTENT_PROJECT_ID`. This helps us identify and organization benchmark related env vars.

Reply:
For sure!

Reply:
bdefa5a